gvsbuild/projects/librsvg.py

```diff
@@ -36,29 +36,27 @@ def __init__(self):
                 "libxml2",
                 "freetype",
             ],
-            patches=[],
+            patches=[
+                # https://gitlab.gnome.org/GNOME/librsvg/-/merge_requests/1040
+                "001-fix-failed-to-rename-query-rust.patch",
+            ],
         )
-        if Project.opts.enable_gi:
-            self.add_dependency("gobject-introspection")
-
-    def build(self):
-        self.builder.mod_env("INCLUDE", "include\\cairo", add_gtk=True)
 
-        b_dir = f"{self.builder.working_dir}\\{self.name}\\win32"
-
-        config = self.builder.opts.configuration
-        gtk_dir = self.builder.gtk_dir
-        rust_ver = Project.get_project("cargo").version
-        python = sys.executable
-        cmd = f'nmake -f makefile.vc CFG={config} "PREFIX={gtk_dir}" CARGO=cargo RUSTUP=rustup "PYTHON={python}" TOOLCHAIN_VERSION={rust_ver} install'
-
-        if Project.opts.enable_gi:
-            cmd += " INTROSPECTION=1"
+        if self.opts.enable_gi:
+            self.add_dependency("gobject-introspection")
+            enable_gi = "enabled"
+        else:
+            enable_gi = "disabled"
 
-        self.push_location(b_dir)
-        self.exec_vs(cmd)
-        self.pop_location()
+        self.add_param(f"-Dintrospection={enable_gi}")
+        self.add_param("-Ddocs=disabled")
+        self.add_param("-Dtests=false")
+        self.add_param("-Dvala=disabled")
 
+    def build(self):
+        # Install cargo-c version compatible with Cargo 0.77.x
```

Comment:
is this still needed if now we can use the latest Rust compiler?

Reply:
Hi @pbor, good catch. We still need to install cargo-c, but I unpinned this older version.